tensorflow/python/kernel_tests/morphological_ops_test.py

```diff
@@ -140,7 +140,7 @@ def _testDilationSamePaddingRate(self, use_gpu):
     image = [[[[.1], [.2], [.3]], [[.4], [.5], [.6]], [[.7], [.8], [.9]]]]
     # [2, 2, 1]
     kernel = [[[.4], [.3]], [[.1], [.2]]]
-    # Because rate = 2, the effective kernel is [3, 3, 1]:
+    # Because rate = 2.0, the effective kernel is [3, 3, 1]:
```

Comment:
Question on consistency: sometimes you use 1., other times 2.0 - should we settle on a single version? There's also the .0 corner case, but that's rare.

Reply:
@mdanatg Here I used `.0` because of the succeeding comma (future-proofing lints / spell-checkers)

Reply:
I see. If at all practical, using .0 in the code as well would sound good to me. No worries if that cannot be easily controlled though - it's just a minor consistency issue.